src/trace-plugin-interface.js

```diff
@@ -102,7 +102,6 @@ RootSpan.prototype.getTraceContext = function() {
 /**
  * PluginAPI constructor. Don't call directly - a plugin object will be passed to
  * plugin themselves
- * TODO(kjin): Should be called something else
```

Comment:
I'm going to hold off landing this until the `PluginAPI` refactoring is done. Then I'll rename to `TraceAPI` as discussed in person.